### PR TITLE
working on #777

### LINKED ISSUE #777: all-but-illegible text style needs fix for humans
Brought to eye by #775, the `<span class="color-text" style="color:#b4a7d6;">` and similar [here in the source](https://github.com/w3c/vc-data-model/blob/6c429cfc54ebfa6f5b74a95dd51d46eed7856b9e/index.html#L834-L843), and [here in the preview (describing figures 7 and 8)](https://w3c.github.io/vc-data-model/#basic-vp) is all-but-illegible in my browser (Chrome Version 90.0.4430.93 (Official Build) (x86_64) on macOS El Capitan (10.11.6)). I does't seem likely that it would be much more legible in any other browser on any other platform.

illegible_text

Regrettably, this styling didn't go through W3 CSS, so it probably counts as an errata for W3 purposes.

I *am* surprised it passed muster for the Accessibility review!

### PR DESCRIPTION
**Pull request by TallTed**
tries to partially (diagram changes are also needed) fix #777, by uncoloring and adjusting the text

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/785.html" title="Last updated on Aug 11, 2021, 3:17 PM UTC (bd8092d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/785/b1ea54d...bd8092d.html" title="Last updated on Aug 11, 2021, 3:17 PM UTC (bd8092d)">Diff</a>

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]

**Comment by kdenhartog** [comment]
Thanks for the PR for this fix. Looks good to me, I'll include it in the round of updates for the CCG to review.

**Comment by kdenhartog** [comment]
@TallTed I had to repoint this at v1.1 for the updates which brought that README.md update back in. Once we get the other PR through then that should get removed I believe.

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by msporny — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -827,21 +827,21 @@ <h3>Presentations</h3>
...
+<a>Graph</a>, expresses the <a>verifiable presentation</a> itself, which
+contains presentation metadata. The <code>verifiableCredential</code>
+property in the Presentation <a>Graph</a> refers to one or more <a>verifiable
+credentials</a>, each being one of the second information <a>graphs</a>, i.e.,
+a self-contained Credential <a>Graph</a>), which in turn contains credential
+metadata and claims. The third information <a>graph</a>, the Credential Proof
+<a>Graph</a>, expresses the credential graph proof, which is usually a digital
+signature. The fourth information <a>graph</a>, the Presentation Proof
+<a>Graph</a>, expresses the presentation graph proof, which is usually a
+digital signature.
```
Comment (TallTed) [inline index.html:844]:
```suggestion
information <a>graphs</a>. The first of these information <a>graphs</a>, the
Presentation <a>Graph</a>, expresses the <a>verifiable presentation</a>
itself, which contains presentation metadata. The
<code>verifiableCredential</code> property in the Presentation <a>Graph</a>
refers to one or more <a>verifiable credentials</a>, each being one of the
second information <a>graphs</a>, i.e., a self-contained Credential
<a>Graph</a>), which in turn contains credential metadata and claims. The
third information <a>graph</a>, the Credential Proof <a>Graph</a>, expresses
the credential graph proof, which is usually a digital signature. The fourth
information <a>graph</a>, the Presentation Proof <a>Graph</a>, expresses the
presentation graph proof, which is usually a digital signature.
```

File: README.md [hunk 2]
```diff
@@ -29,3 +29,33 @@ mailing list as well.
...
+  <dt>Editorial</dt>
+  <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
+  <dt>Substantive</dt>
+  <dd>Mark with "substantial" tag. Bugfixes are merged into separate branch "v1.2". New Features stay around as an open PR.</dd>
+</dl>
+3. W3C CCG is notified of PRs that will be merged in the next 14 days if there
+are no objections.
+4. When it's determined a new reccomendation should go out, the W3C Verifiable
+Credentials Working Group members meet, review all the PRs that have been
+merged, and make a formal recommendation if agreement is reached.
```
Comment (TallTed) [inline README.md:48]:
```suggestion
1. For now, we will focus only on merging new errata PRs into this repository,
   but encourage activity related to new features.
2. Once a PR is opened, chairs and editors make judgement call on whether
   changes are substantive or editorial.
   <dl>
     <dt>Editorial</dt>
     <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
     <dt>Substantive</dt>
     <dd>Mark with "substantial" tag. Bug fixes are merged into separate branch "v1.2". New Features stay around as an open PR.</dd>
   </dl>
3. W3C CCG is notified of PRs that will be merged in the next 14 days if there
   are no objections.
4. When it's determined a new recommendation should go out, the W3C Verifiable
   Credentials Working Group members meet, review all the PRs that have been
   merged, and make a formal recommendation if agreement is reached.
```

File: README.md [hunk 3]
```diff
@@ -29,3 +29,33 @@ mailing list as well.
...
+
+### Roadmap for 2021
+- 1 editorial update (v1.1?)
+- 1 substantive update (v1.2?)
```
Comment (TallTed) [inline README.md:52]:
```suggestion
- one editorial update (v1.1?)
- one substantive update (v1.2?)
```

**Comment by TallTed** [comment]
Sorry this blends `index.html` with `README.md` edits...  I'm hoping that can be OK for this PR, but will create two new PRs if need be.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-08-11-vcwg#section3-4) on 2021-08-11

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.4. working on #777 (pr vc-data-model#785)

_See github pull request [#785](https://github.com/w3c/vc-data-model/pull/785)._




**Ted Thibodeau Jr.:** this is self explanatory. Just did a bit of patch work. Changes readme and spec html  

**Wayne Chang:** we will merge as soon as we hit the 14-day mark  

> *Wayne Chang:* sorry q?

**Kyle Den Hartog:** I'm going to take Ted's readme changes and merge them into main as well  

> *Brent Zundel:* +1 to kdenhartog

> *Manu Sporny:* +1 to merge

</details>

**Comment by kdenhartog** [comment]
multiple positive reviews, feedback received and incorporated, merging

**Comment by kdenhartog** [comment]
There is an Errata label for this on a linked issue, so removing the label from the PR. Also noting that this isn't currently on the V1.1 branch and investigation still needs to be done on what happened here.

**Pull request merged**